spyder/config/main.py

```diff
@@ -94,7 +94,6 @@
               'use_custom_margin': True,
               'custom_margin': 0,
               'use_custom_cursor_blinking': False,
-              'custom_cursor_blinking': 1000,

```

Comment:
If I do that, then delete my .init files and restart Spyder, I get the following traceback:
```
Traceback (most recent call last):
  File "C:\Users\jsgosselin\spyder\spyder\app\mainwindow.py", line 2585, in edit_preferences
    widget.initialize()
  File "C:\Users\jsgosselin\spyder\spyder\plugins\configdialog.py", line 76, in initialize
    self.load_from_conf()
  File "C:\Users\jsgosselin\spyder\spyder\plugins\configdialog.py", line 332, in load_from_conf
    spinbox.setValue(self.get_option(option, default))
TypeError: setValue(self, int): argument 1 has unexpected type 'NoneType'
```